resources/processor-types/custom.yaml

```
@@ -4,14 +4,18 @@ metadata:
   name: custom
   displayName: Custom
   description: Insert a custom OpenTelemetry processor configuration.
+
 spec:
   version: 0.0.1
   parameters:
     - name: configuration
       label: Configuration
-      description: Enter any OpenTelemetry Processor and the YAML will be inserted into the configuration. Processor configuration documentation can be found here https://github.com/observIQ/observiq-otel-collector/blob/main/docs/processors.md.
+      description: Enter any OpenTelemetry Processor and the YAML will be inserted into the configuration.
```

Not sure about this.  I think the OIQ collector has a few more than the otel collector.

Just an Idea - but maybe @ryancgoins can weigh in.

```suggestion
      description: Enter any Supported Processor and the YAML will be inserted into the configuration.
```